OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -29,14 +29,19 @@ public virtual void Init(ModData modData, Dictionary<string, string> info)
 			ModData = modData;
 		}
 
+		bool initialized;
```

Comment:
Can you please define this with the other fields at the top of the class?

Reply:
Okay.